winsup/cygwin/math/nextafterl.c

```diff
@@ -13,9 +13,14 @@
 
 #include <math.h>
 
+double nexttoward(double x, long double y);
```

Comment:
The `nexttoward` forward declaration is in `math.h` (conditioned). Shouldn't rather be the `math.h` modified instead of forward declaration here?